Tests/SonarScanner.MSBuild.Common.Test/ILoggerTests.cs

```diff
@@ -349,23 +350,6 @@ public void ConsoleLogger_WriteUIWarnings_GenerateFile()
              """);
     }
 
-    [TestMethod]
-    public void ConsoleLogger_WriteUIWarnings_ContainsTimeStampOnce()
-    {
-        const string prefixRegex = @"\d{2}:\d{2}:\d{2}(.\d{1,3})?  WARNING:";
-        using var output = new OutputCaptureScope();
-        var logger = new ConsoleLogger(includeTimestamp: true, fileWrapper);
-
-        logger.LogUIWarning("uiWarn1");
-        output.AssertExpectedLastMessageRegex($"{prefixRegex} uiWarn1");
-
-        logger.LogUIWarning("uiWarn2", null);
-        output.AssertExpectedLastMessageRegex($"{prefixRegex} uiWarn2");
-
-        logger.LogUIWarning("uiWarn3 {0}", "xxx");
-        output.AssertExpectedLastMessageRegex($"{prefixRegex} uiWarn3 xxx");
-    }
```

Comment:
This test existed as the friend of the one above it, the only difference being the `includeTimestamp` flag.

Since I can assert the file now, which does no longer contain timestamps, ever, I just merged the two tests.